templ_enum.cif

```diff
@@ -815,6 +815,8 @@ save_units_code
    'seconds'          "time      'seconds'"
    'microseconds'     "time      'microseconds'"
 
+   'reciprocal_seconds'          "Rate      'per second'"
```

Comment:
Maybe it should be called "hertzs" since that is the derivative SI unit?

Reply:
OK, so while commenting on another issue, I stumbled upon a related definition in the mmCIF dictionary [1]:
```
hertz                           "reciprocal seconds"
```

So I would say that for the sake of compatibility we should definitely go with `hertz` as the unit code and not `reciprocal_seconds`.

https://mmcif.wwpdb.org/dictionaries/ascii/mmcif_pdbx_v50.dic

Reply:
Can do. I was just following previous convention.